### ISSUE TITLE
DEVEXP-14: Liberty integration with PostgreSQL

### ISSUE THREAD

**Issue by njr-11** [OpenLiberty/open-liberty#4996]
Built-in integration with PostgreSQL.
This includes work such as

- recognizing the driver JARs and data source classes,
- include a properties.postgreSQL configuration element including known property names/values for data sources,
- ensuring that connections and statements are cleaned up properly by the pool
- other general investigation into ensuring the driver works seamlessly with Liberty

**Comment by marikaj123** [comment]
Beta blog will be provided.

**Comment by marikaj123** [comment]
@aguibert - Please ensure not to move the feature t GA before receiving all of the appropriate focal point approvals.

**Comment by marikaj123** [comment]
Please provide the link to the AHA feature and add the label either 'WAD Approved' or 'No WAD approved'.

**Comment by marikaj123** [comment]
@aguibert - Does the feature require Globalization/Translation?

**Comment by skasund** [comment]
L2 has requested STE slides for this feature.  The STE template can be found at the link below: 

https://ibm.box.com/s/1an42g7zdgmaj84w7dft0indqfgi8ffm

Please upload the completed slides to the same STE Archive BOX folder.  Thanks!

**Comment by gscottj** [comment]
No changes or updates to UI that requires accessibility verification testing.

**Comment by marikaj123** [comment]
According to Andy, the returned translated files are in the stream; therefore, approving.

**Comment by donbourne** [comment]
**Serviceability Approval Comment** - Please answer the following questions for serviceability approval:

1.  WAD -- does the WAD identify the most likely problems customers will see and identify how the feature will enable them to diagnose and solve those problems without resorting to raising a PMR?  Have these issues been addressed in the implementation?

2.  Test and Demo -- As part of the serviceability process we're asking feature teams to test and analyze common problem paths for serviceability and demo those problem paths to someone not involved in the development of the feature (eg. L2, test team, or another development team).  
a) What problem paths were tested and demonstrated? 
b) Who did you demo to?
c) Do the people you demo'd to agree that the serviceability of the demonstrated problem scenarios is sufficient to avoid PMRs for any problems customers are likely to encounter, or that L2 should be able to quickly address those problems without need to engage L3?

3.  SVT -- SVT team is often the first team to try new features and often encounters problems setting up and using them.  Note that we're not expecting SVT to do full serviceability testing -- just to sign-off on the serviceability of the problem paths they encountered.
a) Who conducted SVT tests for this feature?
b) Do they agree that the serviceability of the problems they encountered is sufficient to avoid PMRs, or that L2 should be able to quickly address those problems without need to engage L3?  

4.  Which L2 / L3 queues will handle PMRs for this feature?  Ensure they are present in the contact reference file and in the queue contact summary, and that the respective L2/L3 teams know they are supporting it.  Ask Don Bourne if you need links or more info.

**Comment by aguibert** [comment]
1. This feature has no-WAD approval
2. No demo was done for this feature, and I don't think there would be any meaningful error path scenario to demo. The deliverable here is just additional metatype which is passed through to the JDBC driver, where all vendor-specific validation of DataSource properties is done.
3. This feature did not require SVT testing
4. This github feature is a new metatype element under the existing JDBC features, so the PMRs for this function will go to the same place where other JDBC related PMRs go.

**Comment by skasund** [comment]
@aguibert  STE slides are no longer needed.  I've provided the STE approval.

**Comment by chirp1** [comment]
Approving as Andy made doc updates here: https://github.ibm.com/was-liberty/liberty-docs/pull/1325

**Comment by dave-waddling** [comment]
**FAT Focal Approval comment:** Hi @aguibert, as per my update on the Feature Test Summary (#8118)  I'm adding the FAT Focal Approval.

**Comment by marikaj123** [comment]
@njr-11 - Was the feature code enabled for release 19007 and included into the GM build?

**Comment by njr-11** [comment]
@marikaj123 yes, the commit to switch from beta to ga is showing up as gm-19.0.0.7

**Comment by marikaj123** [comment]
When you get a chance please close the feature.
